canonical-websites/microk8s.io
# Includes Docs at /docs

## Done

Includes Docs to be shown at /docs. Header taken from #30.

## QA

    View the site at https://microk8s-io-canonical-websites-pr-31.run.demo.haus/docs/
    Make sure that the docs are presented as they should be.
 
## Issues
#26 
#27 
#28
#29
